instructors.html

```diff
@@ -27,21 +27,45 @@
         <div class="span10 offset1">
           <h1 class="title">Version Control with Git</h1>
           <h2 class="subtitle">Instructor's Guide</h2>
-<h2 id="legend">Legend</h2>
-<p>Wolfman and Dracula will work together in one project.</p>
-<h2 id="overall">Overall</h2>
+<p>Version control might be the most important topic we teach, but Git is definitely the most complicated tool. However, GitHub presently dominates the open software forge landscape, so we have to help novices learn just enough Git to feel they can and should learn more on their own.</p>
+<p>This is why we don't teach branching: while it is a power tool in the hands of a knowledgeable user, it is an extra cognitive burden for someone who is new to the idea of version control. This is also why we don't get into hashes and commit objects with novices, but try instead to convince them that version control is:</p>
```

Comment:
Suggested rewording:
We don't introduce branching in the novice lesson. While branching extends the power and flexibility of collaborating with others, branching concepts and the variety of possible workflows adds too much cognitive burden for someone who is new to the idea of version control. For similar reasons, we don't get into hashes and commit objects with novices, but try instead to convince them that version control is: